modules/jetty-http-server/src/main/java/com/spotify/apollo/http/server/ApolloRequestHandler.java

```diff
@@ -88,6 +96,15 @@ public void handle(
     baseRequest.setHandled(true);
   }
 
+  private Map<String, String> extractMetadata(HttpServletRequest req) {
+    return ImmutableMap.of(
+        JettyHttpRequestMetadata.PROTOCOL_VERSION.name(), req.getProtocol(),
+        JettyHttpRequestMetadata.REMOTE_ADDRESS.name(), req.getRemoteAddr(),
+        JettyHttpRequestMetadata.REMOTE_PORT.name(), String.valueOf(req.getRemotePort()),
+        RequestMetadata.METADATA_SOURCE, this.getClass().getName()
```

Comment:
No, it could be added anywhere - in the Jetty implementation, it's only added in one place for now.